cli/cmd/env/configure.go

```diff
@@ -118,12 +119,16 @@ func ConfigureCmd(cfg *config.Config) *cobra.Command {
 	return configureCommand
 }
 
-func VariablesFlow(ctx context.Context, projectPath string) (map[string]string, error) {
+func VariablesFlow(ctx context.Context, projectPath string, tel *telemetry.Telemetry) (map[string]string, error) {
 	connectors, err := rillv1beta.ExtractConnectors(ctx, projectPath)
 	if err != nil {
 		return nil, fmt.Errorf("failed to extract connectors %w", err)
 	}
 
+	if tel != nil {
+		tel.EmitDataAccessConnectedStart()
+	}
```

Comment:
Can remove the `if` check here, and it into `tel.emitBehaviourEvent` so it's applied everywhere